### ISSUE TITLE
Support for tabBar.barStyle, navigationBar.barStyle and UIActivityIndicatorViewStyle?

### ISSUE THREAD

**Issue by n3d1117** [jiecao-fm/SwiftTheme#21]
Hi, great library.
In AppDelegate I need to switch between [.default, .black] for both tabBar.barStyle and navigationBar.barStyle. Also [.gray, .white] for UIActivityIndicatorViewStyle. Can you add support for these?
Thanks!

**Comment by wxxsw** [comment]
I added the properties you need, `theme_activityIndicatorViewStyle` and `theme_barStyle`. 

Here are some ways to use it:
```swift
someBar.theme_barStyle = [.default, .black]
someBar.theme_barStyle = "somePlistKeyPath"

activityIndicatorView.theme_activityIndicatorViewStyle = [.white, .whiteLarge, .gray]
activityIndicatorView.theme_activityIndicatorViewStyle = "somePlistKeyPath"
```
Objective-C:
```objective-c
someBar.theme_barStyle = [ThemeBarStylePicker pickerWithStringStyles:@[@"Default", @"Black"]];
someBar.theme_barStyle = [ThemeBarStylePicker pickerWithKeyPath:@"somePlistKeyPath"];

activityIndicatorView.theme_activityIndicatorViewStyle = [ThemeActivityIndicatorViewStylePicker pickerWithStringStyles:@[@"White", @"WhiteLarge", @"Gray"]];
activityIndicatorView.theme_activityIndicatorViewStyle = [ThemeActivityIndicatorViewStylePicker pickerWithKeyPath:@"somePlistKeyPath"];

// Note: StringStyle is not case sensitive.
```

Now has released a new version of [0.3.2](https://github.com/jiecao-fm/SwiftTheme/tree/0.3.2).

**Comment by n3d1117** [comment]
Thank you very much! Works great.
